products/bash/release-mq.sh

```diff
@@ -179,11 +192,11 @@ EOF
         echo -e "\nINFO: For MQ demo pod '$eachMQPod':"
         imageInPod=$(oc get pod $eachMQPod -n $namespace -o json | ./jq -r '.spec.containers[0].image')
         echo "INFO: Image present in the pod '$eachMQPod' is '$imageInPod'"
-        if [[ $imageInPod =~ "$imageTag-test" ]]; then
+        if [[ $imageInPod =~ "$imageTag" ]]; then
```

Comment:
I think this is just checking if $imageInPod contains the $imageTag string anywhere in it? That probably works fine for random tags, but what if the tag is less unique, such as `mq`? Could it instead make sure $imageInPod ends in ':$imageTag', note the ':'

Reply:
Done